### ISSUE TITLE
Gitlab `API V3 is no longer supported. Use API V4 instead.`

### ISSUE THREAD

**Issue by carlitux** [ralphbean/bugwarrior#583]
```bash
ERROR:bugwarrior.services:Worker for [gitlab] failed: Non-200 status code 410; u'https://gitlab.com/api/v3/projects?per_page=100&page=1'; u'{"error":"API V3 is no longer supported. Use API V4 instead."}'
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/bugwarrior/services/__init__.py", line 506, in _aggregate_issues
    for issue in service.issues():
  File "/usr/local/lib/python2.7/site-packages/bugwarrior/services/gitlab.py", line 402, in issues
    all_repos = self._fetch_paged(tmpl)
  File "/usr/local/lib/python2.7/site-packages/bugwarrior/services/gitlab.py", line 329, in _fetch_paged
    items = self._fetch(tmpl, params=params)
  File "/usr/local/lib/python2.7/site-packages/bugwarrior/services/gitlab.py", line 318, in _fetch
    return self.json_response(response)
  File "/usr/local/lib/python2.7/site-packages/bugwarrior/services/__init__.py", line 485, in json_response
    response.status_code, response.url, response.text,
IOError: Non-200 status code 410; u'https://gitlab.com/api/v3/projects?per_page=100&page=1'; u'{"error":"API V3 is no longer supported. Use API V4 instead."}'
```

**Comment by carlitux** [comment]
@ralphbean any deadline for update packages?

**Comment by ralphbean** [comment]
Ah, a new release?

@ryneeverett - do you see any problems with cutting a quick release for this?

**Comment by ryneeverett** [comment]
No regressions that I’m aware of. Go for it.

**Comment by ralphbean** [comment]
Thanks.  :)  Should be available as bugwarrior-1.6.0 now.
